example/vectors.dart

```diff
@@ -52,15 +109,26 @@ class Point {
   }
 
   Point operator +(Vector vector) {
-    Point potision = Point(0, 0);
+    Point position = Point(0, 0);
 
-    potision.x = x + vector.x;
-    potision.y = y + vector.y;
+    position.x = x + vector.x;
+    position.y = y + vector.y;
 
-    return potision;
+    return position;
   }
 }
 
+// Subtracting two points / vectors
```

Comment:
"Subtracting" but / signal used. This is very unclear!

Reply:
What I meant we can subtracting  two points or two vectors. What do you think about that: Subtracting two both vectors and points.

Reply:
"Subtracts two points or two vectors."